### PR TITLE
Client side survey

### PR DESCRIPTION
**Pull request by chantal66**
•	user can create a campaign from the client side by clicking the + button.
	•	user can preview the form before submitting
	•	user can navigate back and forward from preview to the input area and vice versa. 
	•	when user clicks back button data is persisted. 
	•	when user clicks cancel button data is cleared. 
	•	form validation is in place, so user knows that it needs to put some text in the fields, the email field contains regex validation. 
	•	when user clicks send button the email or survey in this case, it’s sent successfully. 
	•	after clicking send the user gets redirected to the dashboard.

### CODE REVIEW

File: client/src/components/surveys/SurveyNew.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+class SurveyNew extends Component {
+  //state initialization
+  state = { showFormReview: false };
+
+  renderContent() {
+    if (this.state.showFormReview) {
+      return (
+        <SurveyFormReview
+          onCancel={() => this.setState({ showFormReview: false })}
```
Comment (kat3kasper) [inline client/src/components/surveys/SurveyNew.js:15]: Don't need the above if statement should be able to do `<SurveyFormReview onCancel={() => this.setState({ showFormReview: !this.state.showFormReview })}`

File: client/src/components/surveys/SurveyNew.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    return (
+      <SurveyForm
+        onSurveySubmit={() => this.setState({ showFormReview: true })}
+      />
+    );
+  }
+
+  render() {
+    return <div>{this.renderContent()}</div>;
```
Comment (kat3kasper) [inline client/src/components/surveys/SurveyNew.js:28]: Think this is overkill in render I would just render the above SurveyForm

**Review by kat3kasper — COMMENTED** [review]
looking good

File: client/src/components/surveys/SurveyNew.js (hunk 2 above)
Comment (chantal66) [reply client/src/components/surveys/SurveyNew.js:28]: thank you! I'll come back to this... I'll put it in my TODO's for refactor..

**Pull request merged**